### ISSUE TITLE
`hidden` property for arguments

### ISSUE THREAD

**Issue by erica** [apple/swift-argument-parser#154]
Sometimes arguments are best handled in USAGE without needing their own section as in the following screen-shot.  I would like to be able to hide the ARGUMENTS section for the location hint(s).

![image](https://user-images.githubusercontent.com/32688/82388897-ecf9ff00-99f7-11ea-9350-739024064cbd.png)

**Comment by natecook1000** [comment]
You can use `@Argument(help: .hidden)` to hide an argument from the help screen, but then it isn't visible in the usage string either. Are you looking for the ability to hide just the ARGUMENTS section, but to keep the usage string intact?

**Comment by erica** [comment]
Yes exactly that: hiding the ARGUMENTS section when it does not add helpful information to preserve vertical space.

**Comment by erica** [comment]
Following up: Typically this would involve a single argument (instance or .remaining), such as `file-names`.
